openmoltools/tests/test_openeye.py

```diff
@@ -42,6 +42,15 @@ def test_butanol_keepconfs():
     assert m1.NumConfs() == 1, "This OEMol was created to have a single conformation."
     assert m1.NumAtoms() == 15, "Butanol should have 15 atoms"
 
+@skipIf(not HAVE_OE, "Cannot test openeye module without OpenEye tools.\n" + openeye_exception_message)
```

Comment:
I think this line is the reason my nosetests don't run, but I just copied it from test_butanol_keepconfs.